{
    "active_lock_reason": null,
    "assignee": null,
    "assignees": [],
    "author_association": "CONTRIBUTOR",
    "body": "### Problem\n\nupgrade to 0.11.0 , remote use it with neovide, nvim crash with\n```\n#4  0x00007f98e6e39e96 in __assert_fail () from /lib/x86_64-linux-gnu/libc.so.6\n#5  0x000055a8cbac8128 in buf_signcols_count_range (buf=<optimized out>, row1=2, row2=2, add=0, clear=kTrue) at /home/xwang/project/neovim/src/nvim/decoration.c:1066\n#6  0x000055a8cbb7bb42 in buf_signcols_count_range (clear=kTrue, add=0, row2=<optimized out>, row1=2, buf=0x55a8e16f92d0) at /home/xwang/project/neovim/src/nvim/extmark.c:647\n#7  extmark_splice_impl (buf=0x55a8e16f92d0, start_row=2, start_col=0, start_byte=41, old_row=1, old_col=0, old_byte=1, new_row=0, new_col=0, new_byte=0, undo=kExtmarkUndo)\n    at /home/xwang/project/neovim/src/nvim/extmark.c:584\n#8  0x000055a8cba5bf0e in extmark_splice (undo=kExtmarkUndo, new_byte=0, new_col=0, new_row=<optimized out>, old_byte=1, old_col=0, old_row=1, start_col=0, start_row=2, buf=<optimized out>)\n    at /home/xwang/project/neovim/src/nvim/extmark.c:556\n#9  nvim_buf_set_lines (channel_id=<optimized out>, buffer=<optimized out>, start=<optimized out>, end=<optimized out>, strict_indexing=<optimized out>, replacement=..., arena=<optimized out>, err=0x7ffd015eeae0)\n    at /home/xwang/project/neovim/src/nvim/api/buffer.c:364\n#10 0x000055a8cbbc194e in nlua_api_nvim_buf_set_lines (lstate=0x7f98e70c5380) at /home/xwang/project/neovim/build/src/nvim/auto/lua_api_c_bindings.generated.h:588\n#11 0x000055a8cbe26f66 in lj_BC_FUNCC ()\n#12 0x000055a8cbe13049 in lua_pcall (L=L@entry=0x7f98e70c5380, nargs=nargs@entry=0, nresults=nresults@entry=0, errfunc=errfunc@entry=-2) at lj_api.c:1122\n#13 0x000055a8cbbe5bdc in nlua_pcall (lstate=lstate@entry=0x7f98e70c5380, nargs=nargs@entry=0, nresults=nresults@entry=0) at /home/xwang/project/neovim/src/nvim/lua/executor.c:180\n#14 0x000055a8cbbe6ed8 in nlua_schedule_event (argv=<optimized out>) at /home/xwang/project/neovim/src/nvim/lua/executor.c:379\n#15 0x000055a8cbd09416 in state_handle_k_event () at /home/xwang/project/neovim/src/nvim/state.c:120\n#16 0x000055a8cbc48398 in nv_event (cap=<optimized out>) at /home/xwang/project/neovim/src/nvim/normal.c:6645\n#17 0x000055a8cbc3c54f in normal_execute (state=0x7ffd015eeda0, key=<optimized out>) at /home/xwang/project/neovim/src/nvim/normal.c:1244\n#18 0x000055a8cbd084d3 in state_enter (s=0x7ffd015eeda0) at /home/xwang/project/neovim/src/nvim/state.c:102\n#19 0x000055a8cbc397cc in normal_enter (cmdwin=<optimized out>, noexmode=<optimized out>) at /home/xwang/project/neovim/src/nvim/normal.c:523\n#20 0x000055a8cba530f7 in main (argc=<optimized out>, argv=<optimized out>) at /home/xwang/project/neovim/src/nvim/main.c:651\n```\n\n### Steps to reproduce\n\nnvim running with \n```\nvim --headless --listen 127.0.0.1:3232\n```\nstill haven't reproduce this\n\n### Expected behavior\n\nnot crash\n\n### Nvim version (nvim -v)\n\n0.11.0 (NVIM v0.11.0 Build type: RelWithDebInfo LuaJIT 2.1.1741730670 \n\n### Vim (not Nvim) behaves the same?\n\nhaven't tried\n\n### Operating system/version\n\nubuntu 22.04.5 LTS\n\n### Terminal name/version\n\nremote with neovide\n\n### $TERM environment variable\n\nremote with neovide\n\n### Installation\n\nbuild from repo",
    "closed_at": null,
    "closed_by": {
        "avatar_url": "https://avatars.githubusercontent.com/u/1359421?v=4",
        "events_url": "https://api.github.com/users/justinmk/events{/privacy}",
        "followers_url": "https://api.github.com/users/justinmk/followers",
        "following_url": "https://api.github.com/users/justinmk/following{/other_user}",
        "gists_url": "https://api.github.com/users/justinmk/gists{/gist_id}",
        "gravatar_id": "",
        "html_url": "https://github.com/justinmk",
        "id": 1359421,
        "login": "justinmk",
        "node_id": "MDQ6VXNlcjEzNTk0MjE=",
        "organizations_url": "https://api.github.com/users/justinmk/orgs",
        "received_events_url": "https://api.github.com/users/justinmk/received_events",
        "repos_url": "https://api.github.com/users/justinmk/repos",
        "site_admin": false,
        "starred_url": "https://api.github.com/users/justinmk/starred{/owner}{/repo}",
        "subscriptions_url": "https://api.github.com/users/justinmk/subscriptions",
        "type": "User",
        "url": "https://api.github.com/users/justinmk",
        "user_view_type": "public"
    },
    "comment_data": [
        {
            "author_association": "NONE",
            "body": "I'm experiencing a problem that may be related, so maybe the following information will help debug this.\n\nEdit: My issue is probably more related to #32379\n\n## Minimal config\n```lua\n-- Run this file as `nvim --clean -u minimal.lua`\n\nfor name, url in pairs {\n  ['nvim-lspconfig'] = 'https://github.com/neovim/nvim-lspconfig',\n  ['gitsigns'] = 'https://github.com/lewis6991/gitsigns.nvim',\n} do\n  local install_path = vim.fn.fnamemodify('nvim_issue/' .. name, ':p')\n  if vim.fn.isdirectory(install_path) == 0 then\n    vim.fn.system { 'git', 'clone', '--depth=1', url, install_path }\n  end\n  vim.opt.runtimepath:append(install_path)\nend\n\nrequire('gitsigns').setup{}\n\n-- Configure pylsp to enable the vim.lsp.buf.format function\nrequire('lspconfig').pylsp.setup{\n  cmd = {\"/path/to/pylsp\"}, -- replace with the actual path to `pylsp`\n}\n\nvim.o.signcolumn = 'auto:2'\n```\n\n## Reproduction steps\n1. In a git repository, I have a tracked python file with the following content:\n```python\nsome_very_very_very_long_variable_name = 10\nif some_very_very_very_long_variable_name is not None and some_very_very_very_long_variable_name > 5 - (10 - 6):\n    pass\n```\n\n2. I change some part of the file (e.g., `10` -> `11`), this causes `gitsigns` to show a sign for the modified line\n3. I run `:lua vim.lsp.buf.format()` so the file is reformatted to:\n```python\nsome_very_very_very_long_variable_name = 11\nif (\n    some_very_very_very_long_variable_name is not None\n    and some_very_very_very_long_variable_name > 5 - (10 - 6)\n):\n    pass\n```\n4. When I then press `u` to undo the change, and then press `<c-r>` to redo the change, Neovim shows the following text and crashes:\n```\nnvim: /usr/src/debug/neovim/neovim/src/nvim/decoration.c:1066: buf_signalcols_count_range: Assertion `buf->b_signcols.count[prevwidth - 1] >= 0' failed.\n```\n\n## Nvim version\nThis is the output of `:verbose version`:\n```\nNVIM v0.11.0\nBuild type: RelWithDebInfo\nLuaJIT 2.1.1741730670\n```\n\n## Operating system\nManjaro Linux 25.0.0\n\n## Terminal name/version\nfoot version: 1.21.0 +pgo +ime +graphemes -assertions\n\n## $TERM\ntmux-256color\n\n## Installation\n`pacman -Syu nvim`",
            "created_at": "2025-04-08T14:55:56Z",
            "html_url": "https://github.com/neovim/neovim/issues/33067#issuecomment-2786733340",
            "id": 2786733340,
            "issue_url": "https://api.github.com/repos/neovim/neovim/issues/33067",
            "node_id": "IC_kwDOAPphoM6mGi0c",
            "performed_via_github_app": null,
            "reactions": {
                "+1": 0,
                "-1": 0,
                "confused": 0,
                "eyes": 0,
                "heart": 0,
                "hooray": 0,
                "laugh": 0,
                "rocket": 0,
                "total_count": 0,
                "url": "https://api.github.com/repos/neovim/neovim/issues/comments/2786733340/reactions"
            },
            "updated_at": "2025-04-09T12:16:24Z",
            "url": "https://api.github.com/repos/neovim/neovim/issues/comments/2786733340",
            "user": {
                "avatar_url": "https://avatars.githubusercontent.com/u/17056143?v=4",
                "events_url": "https://api.github.com/users/jakubbortlik/events{/privacy}",
                "followers_url": "https://api.github.com/users/jakubbortlik/followers",
                "following_url": "https://api.github.com/users/jakubbortlik/following{/other_user}",
                "gists_url": "https://api.github.com/users/jakubbortlik/gists{/gist_id}",
                "gravatar_id": "",
                "html_url": "https://github.com/jakubbortlik",
                "id": 17056143,
                "login": "jakubbortlik",
                "node_id": "MDQ6VXNlcjE3MDU2MTQz",
                "organizations_url": "https://api.github.com/users/jakubbortlik/orgs",
                "received_events_url": "https://api.github.com/users/jakubbortlik/received_events",
                "repos_url": "https://api.github.com/users/jakubbortlik/repos",
                "site_admin": false,
                "starred_url": "https://api.github.com/users/jakubbortlik/starred{/owner}{/repo}",
                "subscriptions_url": "https://api.github.com/users/jakubbortlik/subscriptions",
                "type": "User",
                "url": "https://api.github.com/users/jakubbortlik",
                "user_view_type": "public"
            }
        },
        {
            "author_association": "MEMBER",
            "body": "I was able to reproduce when I noticed the linked neogit issue. Hopefully #33410 resolves this.",
            "created_at": "2025-04-09T23:15:01Z",
            "html_url": "https://github.com/neovim/neovim/issues/33067#issuecomment-2791173018",
            "id": 2791173018,
            "issue_url": "https://api.github.com/repos/neovim/neovim/issues/33067",
            "node_id": "IC_kwDOAPphoM6mXeua",
            "performed_via_github_app": null,
            "reactions": {
                "+1": 0,
                "-1": 0,
                "confused": 0,
                "eyes": 0,
                "heart": 1,
                "hooray": 0,
                "laugh": 0,
                "rocket": 0,
                "total_count": 1,
                "url": "https://api.github.com/repos/neovim/neovim/issues/comments/2791173018/reactions"
            },
            "updated_at": "2025-04-09T23:15:01Z",
            "url": "https://api.github.com/repos/neovim/neovim/issues/comments/2791173018",
            "user": {
                "avatar_url": "https://avatars.githubusercontent.com/u/31730729?v=4",
                "events_url": "https://api.github.com/users/luukvbaal/events{/privacy}",
                "followers_url": "https://api.github.com/users/luukvbaal/followers",
                "following_url": "https://api.github.com/users/luukvbaal/following{/other_user}",
                "gists_url": "https://api.github.com/users/luukvbaal/gists{/gist_id}",
                "gravatar_id": "",
                "html_url": "https://github.com/luukvbaal",
                "id": 31730729,
                "login": "luukvbaal",
                "node_id": "MDQ6VXNlcjMxNzMwNzI5",
                "organizations_url": "https://api.github.com/users/luukvbaal/orgs",
                "received_events_url": "https://api.github.com/users/luukvbaal/received_events",
                "repos_url": "https://api.github.com/users/luukvbaal/repos",
                "site_admin": false,
                "starred_url": "https://api.github.com/users/luukvbaal/starred{/owner}{/repo}",
                "subscriptions_url": "https://api.github.com/users/luukvbaal/subscriptions",
                "type": "User",
                "url": "https://api.github.com/users/luukvbaal",
                "user_view_type": "public"
            }
        },
        {
            "author_association": "CONTRIBUTOR",
            "body": "hello , I want to reopen this issue.  with version e4007551 (which already including b8e5fd) I have similar crash with following backtrace\n\n```\n#5  0x0000559e22be5248 in buf_signcols_count_range (buf=<optimized out>, row1=6, row2=7, add=0, clear=kTrue) at /home/xwang/project/neovim/src/nvim/decoration.c:1066\n#6  0x0000559e22c98e22 in buf_signcols_count_range (clear=kTrue, add=0, row2=<optimized out>, row1=6, buf=0x559e4e3822b0) at /home/xwang/project/neovim/src/nvim/extmark.c:583\n#7  extmark_splice_impl (buf=buf@entry=0x559e4e3822b0, start_row=start_row@entry=6, start_col=start_col@entry=0, start_byte=209, old_row=old_row@entry=1, old_col=old_col@entry=0, old_byte=81, new_row=0, new_col=0, new_byte=0, undo=kExtmarkUndo)\n    at /home/xwang/project/neovim/src/nvim/extmark.c:584\n#8  0x0000559e22c99142 in extmark_adjust (buf=0x559e4e3822b0, line1=7, line2=<optimized out>, amount=<optimized out>, amount_after=-1, undo=kExtmarkUndo) at /home/xwang/project/neovim/src/nvim/extmark.c:518\n#9  0x0000559e22bc8f0e in extmark_adjust (undo=kExtmarkUndo, amount_after=-1, amount=2147483647, line2=7, line1=7, buf=<optimized out>) at /home/xwang/project/neovim/src/nvim/extmark.c:493\n#10 deleted_lines_mark (lnum=7, count=1) at /home/xwang/project/neovim/src/nvim/change.c:495\n#11 0x0000559e22d6b53a in op_delete (oap=0x7ffea02b3a08) at /home/xwang/project/neovim/src/nvim/ops.c:1720\n#12 0x0000559e22d7abae in do_pending_operator (cap=0x7ffea02b3a60, old_col=3, gui_yank=false) at /home/xwang/project/neovim/src/nvim/ops.c:6231\n#13 0x0000559e22d59904 in normal_finish_command (s=0x7ffea02b39f0) at /home/xwang/project/neovim/src/nvim/normal.c:987\n#14 normal_execute (state=0x7ffea02b39f0, key=<optimized out>) at /home/xwang/project/neovim/src/nvim/normal.c:1247\n#15 0x0000559e22e258f1 in state_enter (s=0x7ffea02b39f0) at /home/xwang/project/neovim/src/nvim/state.c:102\n#16 0x0000559e22d56b1c in normal_enter (cmdwin=<optimized out>, noexmode=<optimized out>) at /home/xwang/project/neovim/src/nvim/normal.c:523\n#17 0x0000559e22b700f7 in main (argc=<optimized out>, argv=<optimized out>) at /home/xwang/project/neovim/src/nvim/main.c:651\n```",
            "created_at": "2025-04-16T09:46:18Z",
            "html_url": "https://github.com/neovim/neovim/issues/33067#issuecomment-2809019795",
            "id": 2809019795,
            "issue_url": "https://api.github.com/repos/neovim/neovim/issues/33067",
            "node_id": "IC_kwDOAPphoM6nbj2T",
            "performed_via_github_app": null,
            "reactions": {
                "+1": 0,
                "-1": 0,
                "confused": 0,
                "eyes": 0,
                "heart": 0,
                "hooray": 0,
                "laugh": 0,
                "rocket": 0,
                "total_count": 0,
                "url": "https://api.github.com/repos/neovim/neovim/issues/comments/2809019795/reactions"
            },
            "updated_at": "2025-04-16T09:46:18Z",
            "url": "https://api.github.com/repos/neovim/neovim/issues/comments/2809019795",
            "user": {
                "avatar_url": "https://avatars.githubusercontent.com/u/4547108?v=4",
                "events_url": "https://api.github.com/users/comicfans/events{/privacy}",
                "followers_url": "https://api.github.com/users/comicfans/followers",
                "following_url": "https://api.github.com/users/comicfans/following{/other_user}",
                "gists_url": "https://api.github.com/users/comicfans/gists{/gist_id}",
                "gravatar_id": "",
                "html_url": "https://github.com/comicfans",
                "id": 4547108,
                "login": "comicfans",
                "node_id": "MDQ6VXNlcjQ1NDcxMDg=",
                "organizations_url": "https://api.github.com/users/comicfans/orgs",
                "received_events_url": "https://api.github.com/users/comicfans/received_events",
                "repos_url": "https://api.github.com/users/comicfans/repos",
                "site_admin": false,
                "starred_url": "https://api.github.com/users/comicfans/starred{/owner}{/repo}",
                "subscriptions_url": "https://api.github.com/users/comicfans/subscriptions",
                "type": "User",
                "url": "https://api.github.com/users/comicfans",
                "user_view_type": "public"
            }
        },
        {
            "author_association": "MEMBER",
            "body": "Backtrace doesn't help unfortunately, still need a way to reproduce.",
            "created_at": "2025-04-16T09:53:17Z",
            "html_url": "https://github.com/neovim/neovim/issues/33067#issuecomment-2809039461",
            "id": 2809039461,
            "issue_url": "https://api.github.com/repos/neovim/neovim/issues/33067",
            "node_id": "IC_kwDOAPphoM6nbopl",
            "performed_via_github_app": null,
            "reactions": {
                "+1": 0,
                "-1": 0,
                "confused": 0,
                "eyes": 0,
                "heart": 0,
                "hooray": 0,
                "laugh": 0,
                "rocket": 0,
                "total_count": 0,
                "url": "https://api.github.com/repos/neovim/neovim/issues/comments/2809039461/reactions"
            },
            "updated_at": "2025-04-16T09:53:17Z",
            "url": "https://api.github.com/repos/neovim/neovim/issues/comments/2809039461",
            "user": {
                "avatar_url": "https://avatars.githubusercontent.com/u/31730729?v=4",
                "events_url": "https://api.github.com/users/luukvbaal/events{/privacy}",
                "followers_url": "https://api.github.com/users/luukvbaal/followers",
                "following_url": "https://api.github.com/users/luukvbaal/following{/other_user}",
                "gists_url": "https://api.github.com/users/luukvbaal/gists{/gist_id}",
                "gravatar_id": "",
                "html_url": "https://github.com/luukvbaal",
                "id": 31730729,
                "login": "luukvbaal",
                "node_id": "MDQ6VXNlcjMxNzMwNzI5",
                "organizations_url": "https://api.github.com/users/luukvbaal/orgs",
                "received_events_url": "https://api.github.com/users/luukvbaal/received_events",
                "repos_url": "https://api.github.com/users/luukvbaal/repos",
                "site_admin": false,
                "starred_url": "https://api.github.com/users/luukvbaal/starred{/owner}{/repo}",
                "subscriptions_url": "https://api.github.com/users/luukvbaal/subscriptions",
                "type": "User",
                "url": "https://api.github.com/users/luukvbaal",
                "user_view_type": "public"
            }
        },
        {
            "author_association": "NONE",
            "body": "Happens same here and usually when tracking a file with `gitsigns`  and undoing and redoing causes it to crash with the same error provided above.\nMy initial thought was that maybe more than 1 plugin was drawing on the column but that doesn't seem to be reason for the crashes it seems like.",
            "created_at": "2025-04-22T04:19:59Z",
            "html_url": "https://github.com/neovim/neovim/issues/33067#issuecomment-2820021613",
            "id": 2820021613,
            "issue_url": "https://api.github.com/repos/neovim/neovim/issues/33067",
            "node_id": "IC_kwDOAPphoM6oFh1t",
            "performed_via_github_app": null,
            "reactions": {
                "+1": 0,
                "-1": 0,
                "confused": 0,
                "eyes": 0,
                "heart": 0,
                "hooray": 0,
                "laugh": 0,
                "rocket": 0,
                "total_count": 0,
                "url": "https://api.github.com/repos/neovim/neovim/issues/comments/2820021613/reactions"
            },
            "updated_at": "2025-04-23T09:27:19Z",
            "url": "https://api.github.com/repos/neovim/neovim/issues/comments/2820021613",
            "user": {
                "avatar_url": "https://avatars.githubusercontent.com/u/13983258?v=4",
                "events_url": "https://api.github.com/users/redoxahmii/events{/privacy}",
                "followers_url": "https://api.github.com/users/redoxahmii/followers",
                "following_url": "https://api.github.com/users/redoxahmii/following{/other_user}",
                "gists_url": "https://api.github.com/users/redoxahmii/gists{/gist_id}",
                "gravatar_id": "",
                "html_url": "https://github.com/redoxahmii",
                "id": 13983258,
                "login": "redoxahmii",
                "node_id": "MDQ6VXNlcjEzOTgzMjU4",
                "organizations_url": "https://api.github.com/users/redoxahmii/orgs",
                "received_events_url": "https://api.github.com/users/redoxahmii/received_events",
                "repos_url": "https://api.github.com/users/redoxahmii/repos",
                "site_admin": false,
                "starred_url": "https://api.github.com/users/redoxahmii/starred{/owner}{/repo}",
                "subscriptions_url": "https://api.github.com/users/redoxahmii/subscriptions",
                "type": "User",
                "url": "https://api.github.com/users/redoxahmii",
                "user_view_type": "public"
            }
        },
        {
            "author_association": "NONE",
            "body": "In my case, I think it's causing issue because of [Mini.diff](https://github.com/echasnovski/mini.diff). When I switched to gitsigns.nvim, It's not happening.. (Till now).. @echasnovski Do you have any idea about this ?\n\nScreenshot when it crashed:\n\n![Image](https://github.com/user-attachments/assets/0da30e20-655f-49cd-8585-9aadfec86405)",
            "created_at": "2025-04-22T11:00:08Z",
            "html_url": "https://github.com/neovim/neovim/issues/33067#issuecomment-2820958224",
            "id": 2820958224,
            "issue_url": "https://api.github.com/repos/neovim/neovim/issues/33067",
            "node_id": "IC_kwDOAPphoM6oJGgQ",
            "performed_via_github_app": null,
            "reactions": {
                "+1": 0,
                "-1": 0,
                "confused": 0,
                "eyes": 0,
                "heart": 0,
                "hooray": 0,
                "laugh": 0,
                "rocket": 0,
                "total_count": 0,
                "url": "https://api.github.com/repos/neovim/neovim/issues/comments/2820958224/reactions"
            },
            "updated_at": "2025-04-23T09:27:14Z",
            "url": "https://api.github.com/repos/neovim/neovim/issues/comments/2820958224",
            "user": {
                "avatar_url": "https://avatars.githubusercontent.com/u/3833316?v=4",
                "events_url": "https://api.github.com/users/saifulapm/events{/privacy}",
                "followers_url": "https://api.github.com/users/saifulapm/followers",
                "following_url": "https://api.github.com/users/saifulapm/following{/other_user}",
                "gists_url": "https://api.github.com/users/saifulapm/gists{/gist_id}",
                "gravatar_id": "",
                "html_url": "https://github.com/saifulapm",
                "id": 3833316,
                "login": "saifulapm",
                "node_id": "MDQ6VXNlcjM4MzMzMTY=",
                "organizations_url": "https://api.github.com/users/saifulapm/orgs",
                "received_events_url": "https://api.github.com/users/saifulapm/received_events",
                "repos_url": "https://api.github.com/users/saifulapm/repos",
                "site_admin": false,
                "starred_url": "https://api.github.com/users/saifulapm/starred{/owner}{/repo}",
                "subscriptions_url": "https://api.github.com/users/saifulapm/subscriptions",
                "type": "User",
                "url": "https://api.github.com/users/saifulapm",
                "user_view_type": "public"
            }
        },
        {
            "author_association": "MEMBER",
            "body": "> [@echasnovski](https://github.com/echasnovski) Do you have any idea about this ?\n\nI've seen two instances of unexpected behavior from Neovim when using 'mini.diff'. Both of them got \"fixed\" by using the `Release` build type:\n- [Reddit post](https://www.reddit.com/r/neovim/comments/1k0owx9/neovim_crashes_on_undodelete/).\n- [LazyVim issue](https://github.com/LazyVim/LazyVim/issues/5912#issuecomment-2787548493).\n\nUnfortunately, I can not reproduce any of them (mostly because I prefer to use `Release` build type of 0.11).",
            "created_at": "2025-04-22T12:20:05Z",
            "html_url": "https://github.com/neovim/neovim/issues/33067#issuecomment-2821153763",
            "id": 2821153763,
            "issue_url": "https://api.github.com/repos/neovim/neovim/issues/33067",
            "node_id": "IC_kwDOAPphoM6oJ2Pj",
            "performed_via_github_app": null,
            "reactions": {
                "+1": 0,
                "-1": 0,
                "confused": 0,
                "eyes": 0,
                "heart": 0,
                "hooray": 0,
                "laugh": 0,
                "rocket": 0,
                "total_count": 0,
                "url": "https://api.github.com/repos/neovim/neovim/issues/comments/2821153763/reactions"
            },
            "updated_at": "2025-04-23T09:27:05Z",
            "url": "https://api.github.com/repos/neovim/neovim/issues/comments/2821153763",
            "user": {
                "avatar_url": "https://avatars.githubusercontent.com/u/24854248?v=4",
                "events_url": "https://api.github.com/users/echasnovski/events{/privacy}",
                "followers_url": "https://api.github.com/users/echasnovski/followers",
                "following_url": "https://api.github.com/users/echasnovski/following{/other_user}",
                "gists_url": "https://api.github.com/users/echasnovski/gists{/gist_id}",
                "gravatar_id": "",
                "html_url": "https://github.com/echasnovski",
                "id": 24854248,
                "login": "echasnovski",
                "node_id": "MDQ6VXNlcjI0ODU0MjQ4",
                "organizations_url": "https://api.github.com/users/echasnovski/orgs",
                "received_events_url": "https://api.github.com/users/echasnovski/received_events",
                "repos_url": "https://api.github.com/users/echasnovski/repos",
                "site_admin": false,
                "starred_url": "https://api.github.com/users/echasnovski/starred{/owner}{/repo}",
                "subscriptions_url": "https://api.github.com/users/echasnovski/subscriptions",
                "type": "User",
                "url": "https://api.github.com/users/echasnovski",
                "user_view_type": "public"
            }
        },
        {
            "author_association": "NONE",
            "body": "> > [@echasnovski](https://github.com/echasnovski) Do you have any idea about this ?\n> \n> I've seen two instances of unexpected behavior from Neovim when using 'mini.diff'. Both of them got \"fixed\" by using the `Release` build type:\n> - [Reddit post](https://www.reddit.com/r/neovim/comments/1k0owx9/neovim_crashes_on_undodelete/).\n> - [LazyVim issue](https://github.com/LazyVim/LazyVim/issues/5912#issuecomment-2787548493).\n> \n> Unfortunately, I can not reproduce any of them (mostly because I prefer to use `Release` build type of 0.11).\n\nBTW I am using stable version.. ",
            "created_at": "2025-04-22T12:22:37Z",
            "html_url": "https://github.com/neovim/neovim/issues/33067#issuecomment-2821160289",
            "id": 2821160289,
            "issue_url": "https://api.github.com/repos/neovim/neovim/issues/33067",
            "node_id": "IC_kwDOAPphoM6oJ31h",
            "performed_via_github_app": null,
            "reactions": {
                "+1": 0,
                "-1": 0,
                "confused": 0,
                "eyes": 0,
                "heart": 0,
                "hooray": 0,
                "laugh": 0,
                "rocket": 0,
                "total_count": 0,
                "url": "https://api.github.com/repos/neovim/neovim/issues/comments/2821160289/reactions"
            },
            "updated_at": "2025-04-23T09:27:00Z",
            "url": "https://api.github.com/repos/neovim/neovim/issues/comments/2821160289",
            "user": {
                "avatar_url": "https://avatars.githubusercontent.com/u/3833316?v=4",
                "events_url": "https://api.github.com/users/saifulapm/events{/privacy}",
                "followers_url": "https://api.github.com/users/saifulapm/followers",
                "following_url": "https://api.github.com/users/saifulapm/following{/other_user}",
                "gists_url": "https://api.github.com/users/saifulapm/gists{/gist_id}",
                "gravatar_id": "",
                "html_url": "https://github.com/saifulapm",
                "id": 3833316,
                "login": "saifulapm",
                "node_id": "MDQ6VXNlcjM4MzMzMTY=",
                "organizations_url": "https://api.github.com/users/saifulapm/orgs",
                "received_events_url": "https://api.github.com/users/saifulapm/received_events",
                "repos_url": "https://api.github.com/users/saifulapm/repos",
                "site_admin": false,
                "starred_url": "https://api.github.com/users/saifulapm/starred{/owner}{/repo}",
                "subscriptions_url": "https://api.github.com/users/saifulapm/subscriptions",
                "type": "User",
                "url": "https://api.github.com/users/saifulapm",
                "user_view_type": "public"
            }
        },
        {
            "author_association": "MEMBER",
            "body": "> BTW I am using stable version..\n\nUsing 0.11.0 (a.k.a. \"stable\") version is not enough, as it should be built with `Release` type. This can be not the case, as is shown in the first comment. What is the output of `nvim --version` for you? It should contain `Build type: Release`.",
            "created_at": "2025-04-22T12:34:28Z",
            "html_url": "https://github.com/neovim/neovim/issues/33067#issuecomment-2821189707",
            "id": 2821189707,
            "issue_url": "https://api.github.com/repos/neovim/neovim/issues/33067",
            "node_id": "IC_kwDOAPphoM6oJ_BL",
            "performed_via_github_app": null,
            "reactions": {
                "+1": 0,
                "-1": 0,
                "confused": 0,
                "eyes": 0,
                "heart": 0,
                "hooray": 0,
                "laugh": 0,
                "rocket": 0,
                "total_count": 0,
                "url": "https://api.github.com/repos/neovim/neovim/issues/comments/2821189707/reactions"
            },
            "updated_at": "2025-04-23T09:26:54Z",
            "url": "https://api.github.com/repos/neovim/neovim/issues/comments/2821189707",
            "user": {
                "avatar_url": "https://avatars.githubusercontent.com/u/24854248?v=4",
                "events_url": "https://api.github.com/users/echasnovski/events{/privacy}",
                "followers_url": "https://api.github.com/users/echasnovski/followers",
                "following_url": "https://api.github.com/users/echasnovski/following{/other_user}",
                "gists_url": "https://api.github.com/users/echasnovski/gists{/gist_id}",
                "gravatar_id": "",
                "html_url": "https://github.com/echasnovski",
                "id": 24854248,
                "login": "echasnovski",
                "node_id": "MDQ6VXNlcjI0ODU0MjQ4",
                "organizations_url": "https://api.github.com/users/echasnovski/orgs",
                "received_events_url": "https://api.github.com/users/echasnovski/received_events",
                "repos_url": "https://api.github.com/users/echasnovski/repos",
                "site_admin": false,
                "starred_url": "https://api.github.com/users/echasnovski/starred{/owner}{/repo}",
                "subscriptions_url": "https://api.github.com/users/echasnovski/subscriptions",
                "type": "User",
                "url": "https://api.github.com/users/echasnovski",
                "user_view_type": "public"
            }
        },
        {
            "author_association": "NONE",
            "body": "> > BTW I am using stable version..\n> \n> Using 0.11.0 (a.k.a. \"stable\") version is not enough, as it should be built with `Release` type. This can be not the case, as is shown in the first comment. What is the output of `nvim --version` for you? It should contain `Build type: Release`.\n\n```\nNVIM v0.11.0\nBuild type: RelWithDebInfo\nLuaJIT 2.1.1744318430\nRun \"nvim -V1 -v\" for more info\n````",
            "created_at": "2025-04-23T06:25:55Z",
            "html_url": "https://github.com/neovim/neovim/issues/33067#issuecomment-2823198725",
            "id": 2823198725,
            "issue_url": "https://api.github.com/repos/neovim/neovim/issues/33067",
            "node_id": "IC_kwDOAPphoM6oRpgF",
            "performed_via_github_app": null,
            "reactions": {
                "+1": 0,
                "-1": 0,
                "confused": 0,
                "eyes": 0,
                "heart": 0,
                "hooray": 0,
                "laugh": 0,
                "rocket": 0,
                "total_count": 0,
                "url": "https://api.github.com/repos/neovim/neovim/issues/comments/2823198725/reactions"
            },
            "updated_at": "2025-04-23T09:26:47Z",
            "url": "https://api.github.com/repos/neovim/neovim/issues/comments/2823198725",
            "user": {
                "avatar_url": "https://avatars.githubusercontent.com/u/3833316?v=4",
                "events_url": "https://api.github.com/users/saifulapm/events{/privacy}",
                "followers_url": "https://api.github.com/users/saifulapm/followers",
                "following_url": "https://api.github.com/users/saifulapm/following{/other_user}",
                "gists_url": "https://api.github.com/users/saifulapm/gists{/gist_id}",
                "gravatar_id": "",
                "html_url": "https://github.com/saifulapm",
                "id": 3833316,
                "login": "saifulapm",
                "node_id": "MDQ6VXNlcjM4MzMzMTY=",
                "organizations_url": "https://api.github.com/users/saifulapm/orgs",
                "received_events_url": "https://api.github.com/users/saifulapm/received_events",
                "repos_url": "https://api.github.com/users/saifulapm/repos",
                "site_admin": false,
                "starred_url": "https://api.github.com/users/saifulapm/starred{/owner}{/repo}",
                "subscriptions_url": "https://api.github.com/users/saifulapm/subscriptions",
                "type": "User",
                "url": "https://api.github.com/users/saifulapm",
                "user_view_type": "public"
            }
        },
        {
            "author_association": "MEMBER",
            "body": "I was able to hit the assert from https://github.com/LazyVim/LazyVim/issues/5912#issuecomment-2783876145, and identified some places where the `buf->b_prev_line_count` added in #33410 is left with an incorrect value after inserting/deleting lines without a splice. @bfredl mentioned he might look into if clearing/splicing the marktree is necessary in those places, if not we might reset `b_prev_line_count` manually.",
            "created_at": "2025-04-23T09:26:33Z",
            "html_url": "https://github.com/neovim/neovim/issues/33067#issuecomment-2823655160",
            "id": 2823655160,
            "issue_url": "https://api.github.com/repos/neovim/neovim/issues/33067",
            "node_id": "IC_kwDOAPphoM6oTY74",
            "performed_via_github_app": null,
            "reactions": {
                "+1": 3,
                "-1": 0,
                "confused": 0,
                "eyes": 0,
                "heart": 0,
                "hooray": 0,
                "laugh": 0,
                "rocket": 0,
                "total_count": 3,
                "url": "https://api.github.com/repos/neovim/neovim/issues/comments/2823655160/reactions"
            },
            "updated_at": "2025-04-23T09:26:33Z",
            "url": "https://api.github.com/repos/neovim/neovim/issues/comments/2823655160",
            "user": {
                "avatar_url": "https://avatars.githubusercontent.com/u/31730729?v=4",
                "events_url": "https://api.github.com/users/luukvbaal/events{/privacy}",
                "followers_url": "https://api.github.com/users/luukvbaal/followers",
                "following_url": "https://api.github.com/users/luukvbaal/following{/other_user}",
                "gists_url": "https://api.github.com/users/luukvbaal/gists{/gist_id}",
                "gravatar_id": "",
                "html_url": "https://github.com/luukvbaal",
                "id": 31730729,
                "login": "luukvbaal",
                "node_id": "MDQ6VXNlcjMxNzMwNzI5",
                "organizations_url": "https://api.github.com/users/luukvbaal/orgs",
                "received_events_url": "https://api.github.com/users/luukvbaal/received_events",
                "repos_url": "https://api.github.com/users/luukvbaal/repos",
                "site_admin": false,
                "starred_url": "https://api.github.com/users/luukvbaal/starred{/owner}{/repo}",
                "subscriptions_url": "https://api.github.com/users/luukvbaal/subscriptions",
                "type": "User",
                "url": "https://api.github.com/users/luukvbaal",
                "user_view_type": "public"
            }
        },
        {
            "author_association": "MEMBER",
            "body": "yeah ideally all situations which can result in marks being after the last valid position (after EOL of last line) need to be fixed correctly splice marks, so that invalid positions don't occur and `b_prev_line_count` won't be needed at all. ",
            "created_at": "2025-04-23T10:50:30Z",
            "html_url": "https://github.com/neovim/neovim/issues/33067#issuecomment-2823884794",
            "id": 2823884794,
            "issue_url": "https://api.github.com/repos/neovim/neovim/issues/33067",
            "node_id": "IC_kwDOAPphoM6oUQ_6",
            "performed_via_github_app": null,
            "reactions": {
                "+1": 0,
                "-1": 0,
                "confused": 0,
                "eyes": 0,
                "heart": 0,
                "hooray": 0,
                "laugh": 0,
                "rocket": 0,
                "total_count": 0,
                "url": "https://api.github.com/repos/neovim/neovim/issues/comments/2823884794/reactions"
            },
            "updated_at": "2025-04-23T10:50:30Z",
            "url": "https://api.github.com/repos/neovim/neovim/issues/comments/2823884794",
            "user": {
                "avatar_url": "https://avatars.githubusercontent.com/u/1363104?v=4",
                "events_url": "https://api.github.com/users/bfredl/events{/privacy}",
                "followers_url": "https://api.github.com/users/bfredl/followers",
                "following_url": "https://api.github.com/users/bfredl/following{/other_user}",
                "gists_url": "https://api.github.com/users/bfredl/gists{/gist_id}",
                "gravatar_id": "",
                "html_url": "https://github.com/bfredl",
                "id": 1363104,
                "login": "bfredl",
                "node_id": "MDQ6VXNlcjEzNjMxMDQ=",
                "organizations_url": "https://api.github.com/users/bfredl/orgs",
                "received_events_url": "https://api.github.com/users/bfredl/received_events",
                "repos_url": "https://api.github.com/users/bfredl/repos",
                "site_admin": false,
                "starred_url": "https://api.github.com/users/bfredl/starred{/owner}{/repo}",
                "subscriptions_url": "https://api.github.com/users/bfredl/subscriptions",
                "type": "User",
                "url": "https://api.github.com/users/bfredl",
                "user_view_type": "public"
            }
        },
        {
            "author_association": "NONE",
            "body": "@echasnovski Today It's happening again.. Now I am using gitsign. so I think it's not mini.diff issue anymore..\n\nHere is a video: \n\nhttps://github.com/user-attachments/assets/8225cc24-5110-430f-9fe0-313312f2fb11",
            "created_at": "2025-04-23T16:40:42Z",
            "html_url": "https://github.com/neovim/neovim/issues/33067#issuecomment-2824905347",
            "id": 2824905347,
            "issue_url": "https://api.github.com/repos/neovim/neovim/issues/33067",
            "node_id": "IC_kwDOAPphoM6oYKKD",
            "performed_via_github_app": null,
            "reactions": {
                "+1": 0,
                "-1": 0,
                "confused": 0,
                "eyes": 0,
                "heart": 0,
                "hooray": 0,
                "laugh": 0,
                "rocket": 0,
                "total_count": 0,
                "url": "https://api.github.com/repos/neovim/neovim/issues/comments/2824905347/reactions"
            },
            "updated_at": "2025-04-24T14:39:14Z",
            "url": "https://api.github.com/repos/neovim/neovim/issues/comments/2824905347",
            "user": {
                "avatar_url": "https://avatars.githubusercontent.com/u/3833316?v=4",
                "events_url": "https://api.github.com/users/saifulapm/events{/privacy}",
                "followers_url": "https://api.github.com/users/saifulapm/followers",
                "following_url": "https://api.github.com/users/saifulapm/following{/other_user}",
                "gists_url": "https://api.github.com/users/saifulapm/gists{/gist_id}",
                "gravatar_id": "",
                "html_url": "https://github.com/saifulapm",
                "id": 3833316,
                "login": "saifulapm",
                "node_id": "MDQ6VXNlcjM4MzMzMTY=",
                "organizations_url": "https://api.github.com/users/saifulapm/orgs",
                "received_events_url": "https://api.github.com/users/saifulapm/received_events",
                "repos_url": "https://api.github.com/users/saifulapm/repos",
                "site_admin": false,
                "starred_url": "https://api.github.com/users/saifulapm/starred{/owner}{/repo}",
                "subscriptions_url": "https://api.github.com/users/saifulapm/subscriptions",
                "type": "User",
                "url": "https://api.github.com/users/saifulapm",
                "user_view_type": "public"
            }
        },
        {
            "author_association": "NONE",
            "body": "I have the same issue. It happens mostly when I try to undo changes to my buffer.",
            "created_at": "2025-04-28T16:54:28Z",
            "html_url": "https://github.com/neovim/neovim/issues/33067#issuecomment-2835888951",
            "id": 2835888951,
            "issue_url": "https://api.github.com/repos/neovim/neovim/issues/33067",
            "node_id": "IC_kwDOAPphoM6pCDs3",
            "performed_via_github_app": null,
            "reactions": {
                "+1": 3,
                "-1": 0,
                "confused": 0,
                "eyes": 0,
                "heart": 0,
                "hooray": 0,
                "laugh": 0,
                "rocket": 0,
                "total_count": 3,
                "url": "https://api.github.com/repos/neovim/neovim/issues/comments/2835888951/reactions"
            },
            "updated_at": "2025-04-28T16:57:22Z",
            "url": "https://api.github.com/repos/neovim/neovim/issues/comments/2835888951",
            "user": {
                "avatar_url": "https://avatars.githubusercontent.com/u/55924995?v=4",
                "events_url": "https://api.github.com/users/gzitei/events{/privacy}",
                "followers_url": "https://api.github.com/users/gzitei/followers",
                "following_url": "https://api.github.com/users/gzitei/following{/other_user}",
                "gists_url": "https://api.github.com/users/gzitei/gists{/gist_id}",
                "gravatar_id": "",
                "html_url": "https://github.com/gzitei",
                "id": 55924995,
                "login": "gzitei",
                "node_id": "MDQ6VXNlcjU1OTI0OTk1",
                "organizations_url": "https://api.github.com/users/gzitei/orgs",
                "received_events_url": "https://api.github.com/users/gzitei/received_events",
                "repos_url": "https://api.github.com/users/gzitei/repos",
                "site_admin": false,
                "starred_url": "https://api.github.com/users/gzitei/starred{/owner}{/repo}",
                "subscriptions_url": "https://api.github.com/users/gzitei/subscriptions",
                "type": "User",
                "url": "https://api.github.com/users/gzitei",
                "user_view_type": "public"
            }
        },
        {
            "author_association": "NONE",
            "body": "I know you don't need \"happens for me too\" comments but just wanted to add the info that I am facing this same issue on the current v0.12 nightly (in case that hasn't been confirmed / reprod already). Thanks!\n\nEDIT: It looks like based off this comment (https://github.com/neovim/neovim/issues/34020#issuecomment-2880516285) that if you do the Release build the error does not happen. I will try that and report back, but it was not clear above given the folded/hidden messages so may be good to leave one of the messages expanded/shown here stating Release build is not affected by this. Thanks!",
            "created_at": "2025-05-23T14:02:54Z",
            "html_url": "https://github.com/neovim/neovim/issues/33067#issuecomment-2904534904",
            "id": 2904534904,
            "issue_url": "https://api.github.com/repos/neovim/neovim/issues/33067",
            "node_id": "IC_kwDOAPphoM6tH694",
            "performed_via_github_app": null,
            "reactions": {
                "+1": 0,
                "-1": 0,
                "confused": 0,
                "eyes": 0,
                "heart": 0,
                "hooray": 0,
                "laugh": 0,
                "rocket": 0,
                "total_count": 0,
                "url": "https://api.github.com/repos/neovim/neovim/issues/comments/2904534904/reactions"
            },
            "updated_at": "2025-05-23T14:07:23Z",
            "url": "https://api.github.com/repos/neovim/neovim/issues/comments/2904534904",
            "user": {
                "avatar_url": "https://avatars.githubusercontent.com/u/64155612?v=4",
                "events_url": "https://api.github.com/users/GitMurf/events{/privacy}",
                "followers_url": "https://api.github.com/users/GitMurf/followers",
                "following_url": "https://api.github.com/users/GitMurf/following{/other_user}",
                "gists_url": "https://api.github.com/users/GitMurf/gists{/gist_id}",
                "gravatar_id": "",
                "html_url": "https://github.com/GitMurf",
                "id": 64155612,
                "login": "GitMurf",
                "node_id": "MDQ6VXNlcjY0MTU1NjEy",
                "organizations_url": "https://api.github.com/users/GitMurf/orgs",
                "received_events_url": "https://api.github.com/users/GitMurf/received_events",
                "repos_url": "https://api.github.com/users/GitMurf/repos",
                "site_admin": false,
                "starred_url": "https://api.github.com/users/GitMurf/starred{/owner}{/repo}",
                "subscriptions_url": "https://api.github.com/users/GitMurf/subscriptions",
                "type": "User",
                "url": "https://api.github.com/users/GitMurf",
                "user_view_type": "public"
            }
        },
        {
            "author_association": "CONTRIBUTOR",
            "body": "Just in case this helps @luukvbaal\n\n## 🛠️ Crash Summary\nApp: NVIM v0.12.0-dev-608+gb92e3889fe\nBuild type: RelWithDebInfo\nLuaJIT 2.1.1744317938\n\nmacOS Version: 15.5 (24F74)\n\nArchitecture: ARM-64 (Apple Silicon)\n\nCrash Reason: SIGABRT (Abort trap: 6)\n\n## 💥 Exception Info\nException Type: EXC_CRASH (SIGABRT)\n\nTermination Reason: Namespace SIGNAL, Code 6 Abort trap: 6\n\nCrashed Thread: 0 (Main Thread)\n\n## 📍 Top of Stack (Crash Point)\n\n0   libsystem_kernel.dylib       __pthread_kill\n1   libsystem_pthread.dylib      pthread_kill\n2   libsystem_c.dylib            abort\n3   libsystem_c.dylib            __assert_rtn\n4   nvim                         buf_signcols_count_range + 1112\n5   nvim                         extmark_splice_impl + 368\n6   nvim                         extmark_apply_undo + 808\n7   nvim                         u_undoredo + 2832\n\n## ⚙️ Config init.lua \nHere is the minimal nvim/init.lua that I used to reproduce\n\n```lua\nlocal opt = vim.opt\nopt.fileencoding = \"utf-8\"\nopt.clipboard = \"unnamedplus\"\nopt.mouse = \"\"\n\nopt.backup = false\nopt.undofile = true\nopt.scrolloff = 10\nopt.shell = \"zsh\"\n\nopt.relativenumber = true\nopt.number = true\nopt.numberwidth = 1\nopt.signcolumn = \"yes:1\"\nopt.foldcolumn = \"1\"\nopt.foldlevel = 100\nopt.foldmethod = \"indent\"\n\nfor name, url in pairs({\n  gitsigns = \"https://github.com/lewis6991/gitsigns.nvim\",\n  plenary = \"https://github.com/nvim-lua/plenary.nvim\",\n  statuscol = \"https://github.com/luukvbaal/statuscol.nvim\",\n  lspconfig = \"https://github.com/neovim/nvim-lspconfig\",\n}) do\n  local install_path = vim.fn.fnamemodify(\"gitsigns_issue/\" .. name, \":p\")\n  if vim.fn.isdirectory(install_path) == 0 then\n    vim.fn.system({ \"git\", \"clone\", \"--depth=1\", url, install_path })\n  end\n  vim.opt.runtimepath:append(install_path)\nend\n\n-- Gitsigns setup\nrequire(\"gitsigns\").setup({\n  attach_to_untracked = true,\n})\n\n-- Statuscol integration\nrequire(\"statuscol\").setup({\n  segments = {\n\n    {\n      sign = {\n        namespace = { \"diagnostic*\" },\n        colwidth = 2,\n      },\n    },\n    {\n      text = {\n        require(\"statuscol.builtin\").lnumfunc,\n        \" \",\n      },\n    },\n    {\n      sign = {\n        namespace = { \"gitsign*\" },\n      },\n    },\n    {\n      text = {\n        require(\"statuscol.builtin\").foldfunc,\n        \" \",\n      },\n    },\n    {\n      text = { \" \" },\n    },\n  },\n})\n\n-- Diagnostics and LSP\nvim.diagnostic.config({\n  signs = true,\n  update_in_insert = false,\n})\n\n-- LSP setup \nrequire(\"lspconfig\").lua_ls.setup({\n  cmd = { vim.fn.expand(\"~/.local/share/nvim/mason/bin/lua-language-server\") },\n  settings = {\n    Lua = {\n      runtime = { version = \"LuaJIT\" },\n      diagnostics = { globals = { \"vim\" } },\n      workspace = { library = vim.api.nvim_get_runtime_file(\"\", true) },\n    },\n  },\n})\n\n```\n\nI've been able to consistently trigger this crash by rapidly undoing changes with the above config. While the issue typically occurs only sporadically ( as I mentioned here 👉 [luukvbaal/statuscol.nvim#163](https://github.com/luukvbaal/statuscol.nvim/issues/163)), once it happens, I can usually reproduce it multiple times in a row by repeating the same operation.\n\nThanks @bfredl for this workaround https://github.com/neovim/neovim/pull/34495",
            "created_at": "2025-06-25T08:58:10Z",
            "html_url": "https://github.com/neovim/neovim/issues/33067#issuecomment-3003980103",
            "id": 3003980103,
            "issue_url": "https://api.github.com/repos/neovim/neovim/issues/33067",
            "node_id": "IC_kwDOAPphoM6zDRlH",
            "performed_via_github_app": null,
            "reactions": {
                "+1": 0,
                "-1": 0,
                "confused": 0,
                "eyes": 0,
                "heart": 0,
                "hooray": 0,
                "laugh": 0,
                "rocket": 0,
                "total_count": 0,
                "url": "https://api.github.com/repos/neovim/neovim/issues/comments/3003980103/reactions"
            },
            "updated_at": "2025-06-25T08:58:10Z",
            "url": "https://api.github.com/repos/neovim/neovim/issues/comments/3003980103",
            "user": {
                "avatar_url": "https://avatars.githubusercontent.com/u/3860298?v=4",
                "events_url": "https://api.github.com/users/marcoSven/events{/privacy}",
                "followers_url": "https://api.github.com/users/marcoSven/followers",
                "following_url": "https://api.github.com/users/marcoSven/following{/other_user}",
                "gists_url": "https://api.github.com/users/marcoSven/gists{/gist_id}",
                "gravatar_id": "",
                "html_url": "https://github.com/marcoSven",
                "id": 3860298,
                "login": "marcoSven",
                "node_id": "MDQ6VXNlcjM4NjAyOTg=",
                "organizations_url": "https://api.github.com/users/marcoSven/orgs",
                "received_events_url": "https://api.github.com/users/marcoSven/received_events",
                "repos_url": "https://api.github.com/users/marcoSven/repos",
                "site_admin": false,
                "starred_url": "https://api.github.com/users/marcoSven/starred{/owner}{/repo}",
                "subscriptions_url": "https://api.github.com/users/marcoSven/subscriptions",
                "type": "User",
                "url": "https://api.github.com/users/marcoSven",
                "user_view_type": "public"
            }
        }
    ],
    "comments": 16,
    "comments_url": "https://api.github.com/repos/neovim/neovim/issues/33067/comments",
    "created_at": "2025-03-26T15:14:31Z",
    "events_url": "https://api.github.com/repos/neovim/neovim/issues/33067/events",
    "html_url": "https://github.com/neovim/neovim/issues/33067",
    "id": 2949975954,
    "labels": [
        {
            "color": "FBCA04",
            "default": false,
            "description": "We need minimal steps to reproduce the issue",
            "id": 298863445,
            "name": "needs:repro",
            "node_id": "MDU6TGFiZWwyOTg4NjM0NDU=",
            "url": "https://api.github.com/repos/neovim/neovim/labels/needs:repro"
        },
        {
            "color": "0E8A16",
            "default": false,
            "description": "issue contains a stacktrace/ASAN log",
            "id": 435854079,
            "name": "has:backtrace",
            "node_id": "MDU6TGFiZWw0MzU4NTQwNzk=",
            "url": "https://api.github.com/repos/neovim/neovim/labels/has:backtrace"
        },
        {
            "color": "F9D0C4",
            "default": false,
            "description": "issue reporting a crash or segfault",
            "id": 435854234,
            "name": "bug-crash",
            "node_id": "MDU6TGFiZWw0MzU4NTQyMzQ=",
            "url": "https://api.github.com/repos/neovim/neovim/labels/bug-crash"
        },
        {
            "color": "c5def5",
            "default": false,
            "description": "sign/number column",
            "id": 2962322085,
            "name": "column",
            "node_id": "MDU6TGFiZWwyOTYyMzIyMDg1",
            "url": "https://api.github.com/repos/neovim/neovim/labels/column"
        }
    ],
    "labels_url": "https://api.github.com/repos/neovim/neovim/issues/33067/labels{/name}",
    "locked": false,
    "milestone": null,
    "node_id": "I_kwDOAPphoM6v1Q-S",
    "number": 33067,
    "performed_via_github_app": null,
    "reactions": {
        "+1": 17,
        "-1": 0,
        "confused": 0,
        "eyes": 0,
        "heart": 0,
        "hooray": 0,
        "laugh": 0,
        "rocket": 0,
        "total_count": 17,
        "url": "https://api.github.com/repos/neovim/neovim/issues/33067/reactions"
    },
    "repository_url": "https://api.github.com/repos/neovim/neovim",
    "state": "open",
    "state_reason": "reopened",
    "sub_issues_summary": {
        "completed": 0,
        "percent_completed": 0,
        "total": 0
    },
    "timeline_url": "https://api.github.com/repos/neovim/neovim/issues/33067/timeline",
    "title": "0.11 crash in buf_signcols_count_range",
    "type": {
        "color": "red",
        "created_at": "2024-01-25T10:10:19Z",
        "description": "An unexpected problem or behavior",
        "id": 597163,
        "is_enabled": true,
        "name": "Bug",
        "node_id": "IT_kwDOAGK_Pc4ACRyr",
        "updated_at": "2024-07-26T10:12:30Z"
    },
    "updated_at": "2025-06-25T08:58:10Z",
    "url": "https://api.github.com/repos/neovim/neovim/issues/33067",
    "user": {
        "avatar_url": "https://avatars.githubusercontent.com/u/4547108?v=4",
        "events_url": "https://api.github.com/users/comicfans/events{/privacy}",
        "followers_url": "https://api.github.com/users/comicfans/followers",
        "following_url": "https://api.github.com/users/comicfans/following{/other_user}",
        "gists_url": "https://api.github.com/users/comicfans/gists{/gist_id}",
        "gravatar_id": "",
        "html_url": "https://github.com/comicfans",
        "id": 4547108,
        "login": "comicfans",
        "node_id": "MDQ6VXNlcjQ1NDcxMDg=",
        "organizations_url": "https://api.github.com/users/comicfans/orgs",
        "received_events_url": "https://api.github.com/users/comicfans/received_events",
        "repos_url": "https://api.github.com/users/comicfans/repos",
        "site_admin": false,
        "starred_url": "https://api.github.com/users/comicfans/starred{/owner}{/repo}",
        "subscriptions_url": "https://api.github.com/users/comicfans/subscriptions",
        "type": "User",
        "url": "https://api.github.com/users/comicfans",
        "user_view_type": "public"
    }
}